website/content/en/v0.31/troubleshooting.md

```diff
@@ -619,6 +619,17 @@ This typically occurs when the node has not been considered fully initialized fo
 
 This error indicates that the `vpc.amazonaws.com/pod-eni` resource was never reported on the node. If you've enabled Pod ENI for Karpenter nodes via the `aws.enablePodENI` setting, you will need to make the corresponding change to the VPC CNI to enable [security groups for pods](https://docs.aws.amazon.com/eks/latest/userguide/security-groups-for-pods.html) which will cause the resource to be registered.
 
+### Node provisioning and deprovisioning loop
```

Comment:
I think there is a third option here, which is to use the native interruption handling that Karpenter offers. Should we link users that come here to the "Interruption" section of the disruption doc?

Reply:
I think that option should also come with a caveat - In case a user uses both NTH with SQS and Karpenter with SQS (as stated in the interruption section). It could potentially have the same loop effect.

Reply:
Is using both a real scenario? I think we absolutely need to reference users over to the interruption documentation, since this should be Karpenter's de-facto way for handling these notifications. I'm fine telling users how to do it with NTH since there's definitely a subset of users that are using Karpenter with NTH, but those should be the exception, not the rule

Reply:
I think the only good case for using NTH now is if you need rebalance recommendation draining for nodes

Reply:
I see your point, ill add a reference and some notes as well.